src/features/home/pictures/pictures.tsx

```diff
@@ -38,7 +38,7 @@ const socialMedia: SocialMediaProps[] = [
 ];
 
 export const Pictures = () => (
-  <div className="pictures container">
+  <article className="pictures container">
     <div className="pictures content">
```

Comment:
"Wrapping the component in a single semantic tag is not enough, you need to work with the entire component. There seems to be quite a lot of logic here. 
Why does this **div** not warrant becoming a **section**? 
What is the reason for such nesting? I don't see a necessity for it."